packages/account/src/Sections/Profile/PhoneVerification/otp-verification.tsx

```diff
@@ -15,13 +16,20 @@ type TOTPVerification = {
 
 const OTPVerification = observer(({ phone_verification_type, setOtpVerification }: TOTPVerification) => {
     const { client, ui } = useStore();
-    const { account_settings } = client;
-    const { email, phone } = account_settings;
+    const { account_settings, email } = client;
+    const { phone } = account_settings;
     const [should_show_didnt_get_the_code_modal, setShouldShowDidntGetTheCodeModal] = React.useState(false);
     const [start_timer, setStartTimer] = React.useState(true);
+    const { requestEmailVerificationOTP } = useGetEmailVerificationOTP();
     //TODO: this shall be replace by BE API call when it's ready
     const { should_show_phone_number_otp } = ui;
 
+    React.useEffect(() => {
+        if (!should_show_phone_number_otp) {
+            requestEmailVerificationOTP();
+        }
+    }, [should_show_phone_number_otp, requestEmailVerificationOTP]);
+
     return (
```

Comment:
@suisin-deriv lets add a TODO: on line 71 to remove hardcoded time value once API is ready